src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -117,6 +117,12 @@ void PlatformManagerImpl::WiFIIPChangeListener()
                             continue;
                         }
 
+                        if (ConnectivityManagerImpl::GetWiFiIfName() == nullptr)
+                        {
+                            ChipLogDetail(DeviceLayer, "WiFi device not init. Ignore IP update event");
```

Comment:
Should this be "log detail"?

What does `not init` mean? I think this should be a comment like `This can happen if WiFi is not initialized`. The message however does not seem to check/know that wifi is not initiailized. It only knows `Failed to get wifi interface name` and that should be in the message. Error message should only contain what they know must be true.

Reply:
@andy31415 thank you for your reply. Actually I don't mean to warn the user that the WiFi interface didn't init. I mean the IP update event will ignore. And what I intent to resolve in this patch is not display the warning log. I need to skip the coming "strcmp()" function if the GetWiFiIfName() is nullptr.
This patch fixed one issue I meet:
on one Linux platform, which don't have WiFi interface enabled (no wlan0 and only eth0). Run chip-tool(interactive mode) or chip-lignthing-app.
After about 1 hour, the IP update event is received by the WiFIIPChangeListener(), and the "strcmp()" will compare with the GetWiFiIfName() directly without checking whether it is nullptr. And a Segmentation fault will happen and chip-tool and chip-lighting-app will exit.

Reply:
The log says "wifi device not init". However the check is `GetWiFiIfName == nullptr`.

My complain is that it does not seem guaranteed that `GetWiFiName()` will return null IF and ONLY IF wifi is not initialized (or at least it is very unclear what 'wifi initialized' means. On linux it seems it will be null if `getifaddrs` fails or if none of the found interfaces are wifi.

Re-state this as `No wifi interface name. Ignoring IP update event` since this is what the code is doing.

Reply:
@andy31415 thank you for the comments. I've updated the change and the commit message. Please review.